challenges/tic-tac-toe-C++.md

```diff
@@ -0,0 +1,309 @@
+# Tic -Tac Toe Game
+
+## Description: 
+
+# Rules of the Game
```

Comment:
Maintain the level of headings. As the first one level one, so other levels should be level 2 (##)